static/src/javascripts-legacy/projects/common/modules/atoms/story-questions.js

```diff
@@ -37,7 +37,7 @@ define([
                         askQuestionBtn.classList.add('is-hidden');
                     }
 
-                    if (isEmailSubmissionReady === true) {
+                    if (isEmailSubmissionReady === "true") {
```

Comment:
I am curious, why does strict equals not suffice here?

if the value of `isEmailSubmissionReady` is a boolean, wouldn't the original work?